packages/apps/desktop-viewer-test/package.json

```diff
@@ -0,0 +1,94 @@
+{
+  "name": "@itwin/desktop-viewer-test",
+  "version": "0.1.0",
+  "private": true,
+  "os": [
+    "win32",
+    "linux",
+    "darwin"
+  ],
+  "engines": {
+    "node": ">=10.17.0 <15.0"
+  },
+  "scripts": {
+    "build": "",
+    "build:backend": "tsc -p tsconfig.backend.json",
+    "build:frontend": "cross-env DISABLE_ESLINT=true SKIP_PREFLIGHT_CHECK=true GENERATE_SOURCEMAP=false USE_FAST_SASS=true TRANSPILE_DEPS=false DISABLE_TERSER=true react-scripts build",

```

Comment:
i think itd be better if we moved these env vars from in the script into the .env file
something i wanted to change in core test-apps and desktop-starter too to be more consistent

Reply:
agree. done